tasks_downstream.py

```diff
@@ -388,6 +388,10 @@ def develop(c):
     """Set up a basic development environment."""
     # Prepare environment
     Path(PROJECT_ROOT, "odoo", "auto", "addons").mkdir(parents=True, exist_ok=True)
+    if ODOO_VERSION < 11:
+        Path(PROJECT_ROOT, "odoo", "custom", "build.d", "20-update-pg-repos").chmod(
+            0o755
```

Comment:
This should move somewhere else. It doesn't make sense to keep changing permissions to this file each time we run `inv develop`.

Also, change to `0o750` if possible.

Reply:
Ok, I moved it to a copier task :+1: 

I left the mode as `0750`, as suggested. However, I see that other build files in doodba have `0755`. Could this cause problems when building in other environments?

Reply:
Ok, thanks, I made the chane